src/test/mocks.ts

```diff
@@ -264,7 +264,7 @@ function newTxReceiptResult(
 ): Result {
 	const _acc = acc ? acc : new Account(tx.nonce, new Assets(), new Assets());
 	const delta = new Map<string, Account>([[tx.sender.key, _acc]]);
-	const txr = new TxReceipt(tx, delta);
+	const txr = new TxReceipt(tx, delta, 1);
```

Comment:
Give the function `newTxReceiptResult` a default argument for `TXStatus` and use it here.